src/app/sandbox/components/ContractCall/AvailableFunctionsView.tsx

```diff
@@ -10,7 +11,7 @@ import { Text } from '@/ui/typography';
 import { forwardRef, useColorMode } from '@chakra-ui/react';
 import { mdiApi, mdiFunction } from '@mdi/js';
 import Icon from '@mdi/react';
-import React, { FC } from 'react';
```

Comment:
This seems like an unrelated removal? Can we avoid any unnecessary changes in the PR to simplify the diff? Thanks.

Reply:
this happened when I ran format